### PR TITLE
beta version docs

### PR DESCRIPTION
**Pull request by Maayan-s**

### CODE REVIEW

File: docs/cloud/elementary-managed.mdx [hunk 1]
```diff
@@ -2,11 +2,72 @@
 title: "Elementary managed"
```
Comment (elongl) [inline docs/cloud/elementary-managed.mdx:2]: Maybe upper-case the m? _Elementary Managed_.
Unlike our other pages, this is a name of a product so should be capitalized in my opinion.

File: docs/cloud/elementary-managed.mdx [hunk 2]
```diff
@@ -2,11 +2,72 @@
...
 ---
 
+Elementary managed is a cloud service that offers orchestration and hosting of Elementary as a service.
```
Comment (elongl) [inline docs/cloud/elementary-managed.mdx:5]:
```suggestion
Elementary Managed is a cloud service that offers orchestration and hosting of Elementary.
```

File: docs/cloud/elementary-managed.mdx [hunk 3]
```diff
@@ -1,12 +1,73 @@
 ---
-title: "Elementary managed"
+title: "Elementary Managed"
 ---
 
+Elementary Managed is a cloud service that offers orchestration and hosting of Elementary as a service.
```
Comment (elongl) [inline docs/cloud/elementary-managed.mdx:5]:
```suggestion
Elementary Managed is a cloud service that offers orchestration and hosting of Elementary.
```

File: docs/cloud/elementary-managed.mdx (hunk 1 above)
Comment (Maayan-s) [reply docs/cloud/elementary-managed.mdx]: Yes

**Pull request merged**